### PR TITLE
App manager support for report last sync tile

### PR DESCRIPTION
**Pull request by orangejenny**
##### SUMMARY
https://docs.google.com/document/d/18wnlqyTaRCEmLHcHyS15Ya5oOhjZ9On9dVVH2QXch3s

Builds on https://github.com/dimagi/commcare-hq/pull/26489

##### FEATURE FLAG
Mobile UCR

##### PRODUCT DESCRIPTION

Adds this "Throughout this menu, include a persistent report tile..." checkbox to menu settings. For case and survey menus it appears under "Advanced" while for report menus there's only one settings card.
Screen Shot 2020-02-04 at 11 11 46 AM

Checking the checkbox adds this "Reports last Updated" tile on mobile with the last date report data was synced:
Screenshot_20200204-110807

### CODE REVIEW

**Comment by orangejenny** [comment]
ignoring the lint errors

File: corehq/toggles.py [hunk 1]
```diff
@@ -872,6 +872,13 @@ def _enable_search_index(domain, enabled):
     namespaces=[NAMESPACE_DOMAIN],
 )
 
+MOBILE_UCR_TILE = StaticToggle(
```
Comment (snopoke) [inline corehq/toggles.py]: Is it necessary to have a separate toggle? Why not put this under the existing `MOBILE_UCR` toggle?

**Review by snopoke — APPROVED** [review]
One question about adding a new toggle

File: corehq/toggles.py (hunk 1 above)
Comment (orangejenny) [reply corehq/toggles.py]: sounds reasonable, 39d43e31d7b

Comment (orangejenny) [reply corehq/toggles.py]: sounds reasonable, 39d43e31d7b

**Review by snopoke — APPROVED** [review]

**Review by snopoke — APPROVED** [review]

**Comment by orangejenny** [comment]
Tested on staging; this is working now for survey modules, case modules, and report modules.

**Comment by orangejenny** [comment]
updated description & screenshots

**Review by snopoke — APPROVED** [review]

**Pull request merged**